crates/daphne-server/tests/e2e/test_runner.rs

```diff
@@ -359,11 +359,11 @@ impl TestRunner {
         get_raw_hpke_config(client, self.task_id.as_ref(), &self.helper_url, "helper").await
     }
 
-    #[expect(dead_code)]
-    pub async fn leader_post_expect_ok(
+    pub async fn leader_request_expect_ok(
         &self,
         client: &reqwest::Client,
         path: &str,
+        method: &http::Method,
```

Comment:
Don't use a reference here. I don't think there's much to gain from it, and you end up cloning anyway. So just take by value

Reply:
The borrow checker didn't let me. This was just my hack to make the borrow checker happy :sweat_smile: